src/ifconfig.interfaces.js

```diff
@@ -24,14 +26,22 @@ module.exports = function (cp) {
         if (stderr) {
           return f(stderr);
         }
-
-        f(null, parse(ifConfigOut, routeOut));
+        
+        fs.readFile(interfaces.FILE, {encoding: 'utf-8'}, (error, content) => {
+            if(error) {
+              console.warn(error);
```

Comment:
you know the rule https://www.linuxtopia.org/online_books/programming_books/art_of_unix_programming/ch11s09.html :)

- I think the whole "interfaces" should yield an error if interfaces.FILE could not be read.
- instead of changing the constant `interfaces.FILE` let the user specify a second argument to `interfaces(f)` -> `interfaces(f, options)` (where options default value could be `interface:{file: '/etc/network/interfaces', parse: false}`, to specify the interface file to parse and if the user wants it to be parsed or not.

The test should be updated accordingly but it will give:
- backward compatibility regarding behavior
- extensibility for the futur (the options object can have other parameters)
- user-side choice if he wants the interface file to be parsed or not and if it does, then it can get an error from that file reading/parsing.

Reply:
Added the option